src/main/webapp/app/assessment/assessment-filters/assessment-filters.component.ts

```diff
@@ -13,12 +13,14 @@ enum FilterProp {
     selector: 'jhi-assessment-filters',
     templateUrl: './assessment-filters.component.html',
 })
-export class AssessmentFiltersComponent {
+export class AssessmentFiltersComponent implements OnChanges {
     FilterProp = FilterProp;
 
     filterProp: FilterProp = FilterProp.ALL;
 
-    @Input() submissions: Submission[] = [];
+    @Input()
+    submissions: Submission[] = [];
+    filteredSubmissions: number;
```

Comment:
You should consider renaming this. Currently the name suggests that this attribute contains the actual filtered submissions instead of only the amount of filtered submissions.